### DISCUSSION TITLE
Run/Debug V2

### DISCUSSION THREAD

**Discussion by heribertolugo** [DevToys-app/DevToys#970] [General]
Is there any guide on how to get v2 to build? 

I created one of the features (didnt realize i was in v1 main branch) and then created a branch to implement the feature in v2, but i cannot get it to build or run. I honestly cant even find any of the files which i worked on in the v1 code.

**Comment by veler** [comment]
Hi,
DevToys 2.0 has been rewritten from scratch and the code base looks very different in the branch `dev/2.0`.
I would suggest to look at the CONTRIBUTING.md file on that branch in question: https://github.com/veler/DevToys/blob/dev/2.0/CONTRIBUTING.md#how-to-build-and-run-devtoys-from-source

**Reply by heribertolugo** [reply]
Thanks
